### ISSUE TITLE
MessageAttachment stream - type should be "Readable" not "ReadStream"

### ISSUE THREAD

**Issue by lselden** [eleith/emailjs#261]
in v3.2 the "stream" arg for a MessageAttachment expects fs.ReadStream. But that means other types of streams (not from filesystem) will throw an error. Instead, you should use the type `import {Readable} from "stream"` - or even the lower level `NodeJS.ReadableStream`, which is all that you need to ensure the type has the `.readable` property and can be piped.

#### Relevant line:
https://github.com/eleith/emailjs/blob/v3.2.0/smtp/message.ts#L51

#### Reproduce Error:
```js
import zlib from 'zlib';
import fs from 'fs';
import { MessageAttachment } from 'emailjs';

const attachment: MessageAttachment = {
    // this will throw typescript type error, because createGzip() resolves to Transform type (which extends Readable)
    stream: fs.createReadStream(logpath).pipe(zlib.createGzip())
}
```

**Comment by zackschuster** [comment]
i went with `stream.Readable`, mostly to keep type imports consistent. see `smtp/message: fix attachment stream type` cb692976c6fa3ce83f85770c92ca648c4bd0f073.

thanks for the heads-up :)

**Comment by zackschuster** [comment]
should be fixed as of `3.2.1`.
